ui_buttons.h

```diff
@@ -12,6 +12,7 @@ static void button_setcolors_success(BUTTON *b)
     b->c3 = COLOR_BUTTON_SUCCESS_HOVER_BACKGROUND;
     b->ct1 = COLOR_BUTTON_SUCCESS_TEXT;
     b->ct2 = COLOR_BUTTON_SUCCESS_HOVER_TEXT;
+    b->disabled = 0;
```

Comment:
These functions are meant for colours only (as name suggests). Button colour is a *effect* of being disabled, not an *cause*.